### DISCUSSION TITLE
Initial page load animation

### DISCUSSION THREAD

**Discussion by k16e** [swup/swup#859] [Help] [answered]
### Describe your problem 🧐

Is it possible,... well since Swup is using Swup, like on Swup's on home page,.. to use Swup to animate the very first entry of page/refresh, not only animating between pages after first "unanimated load"? If so how. I don't see it treated anywhere on the Swup docs. 

### What is your question? 🙋‍♀️

How can I use Swup to animate the page's very first load or refresh of same page?

### URL to minimal reproduction 🔗

https://sayings.cc/

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for answers to my question
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

**Comment by hirasso** [comment, marked as answer]
Hi there! Swup is only for page transitions. You could even argue that swups main focus is to progressively convert a multi page app/website into a single page app, with first-class support for animating the page visits. The front page animation on swup.js.org was made with [GSAP](https://github.com/greensock/GSAP).

If you are curious how the animation on swup.js.org was made, [have a look here](https://github.com/swup/docs/blob/master/src/_assets/components/alpine/Intro/Intro.js).

**Reply by k16e** [reply]
Boom! Someone should've said this since 😄. So, I should use gsap then to animate page first/refresh entry. Thanks.

**Reply by hirasso** [reply]
Interesting feedback. I can imagine that you are not alone with your confusion. We'll think about adding a note to the [getting started](https://swup.js.org/getting-started/) page mentioning that the intro animation is NOT related to swup 😅
